src/qtkit.cs

```diff
@@ -614,17 +615,20 @@ interface QTFormatDescription {
 
 	[BaseType (typeof (NSObject))]
 	interface QTMedia {
-		[Availability (Introduced = Platform.Mac_10_3 | Platform.Mac_Arch32, Deprecated = Platform.Mac_10_9)]
+		[IntroducedAttribute (PlatformName.MacOSX, 10, 3, PlatformArchitecture.Arch32)] 
+		[DeprecatedAttribute (PlatformName.MacOSX, 10, 5)] 
```

Comment:
mismatch, deprecated was 10.9 and now is 10.5

Reply:
I copy pasted a few of these before writing up macros, and I apparently screwed a few up...